src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -943,12 +943,25 @@ default Set<ExceptionalDeath> deathSafeExceptions() {
         return EnumSet.of(ExceptionalDeath.FIGHT_CAVE, ExceptionalDeath.INFERNO);
     }
 
+    @ConfigItem(
+        keyName = "deathIgnoredRegions",
+        name = "Ignored Region IDs",
+        description = "User-specified, comma-separated Region IDs where deaths should be ignored.<br/>" +
+            "Use the '::DinkRegion' chat command or an online map to obtain the region IDs.<br/>" +
```

Comment:
```suggestion
            "Use the '::dinkregion' chat command or an online map to obtain the region IDs.<br/>" +
```
command is case-insensitive and our export command readme mentions are lowercase